utils/warnings.ml

```diff
@@ -514,15 +513,35 @@ let letter_alert tokens =
         else
           Format.fprintf ppf "%a%d..%d" print_modifier m a b
     | Letter(l,Some m) -> Format.fprintf ppf "%a%c" print_modifier m l
-    | Letter(l,None) -> print_char ppf l
+    | Letter(l,None) -> print_warning_char ppf l
   in
-  match List.find_map deprecated tokens with
-  | None -> None
-  | Some example ->
+  let consecutive_letters =
+    (* we are tracking consecutive unsigned letter in warning strings:
```

Comment:
apologies for the late-night nitpick (somewhere on earth!), but you never explicitly say that we are considering at-least-2 consecutive letters. Maybe that would be worth pointing out in the comment?

```
We are tracking sequences of 2 or more consecutive unsigned letters in warning strings:
```

Reply:
Is a single letter still consecutive with itself? Anyway, I will disambiguate this comment and add some tests tomorrow (somewhere on Earth).